changelog.d/7260.bugfix

```diff
@@ -0,0 +1 @@
+Fix room publish permissions not being checked on room creation.
```

Comment:
By the way, you can add "Contributed by <your name>." to the end of this if you'd like.

Reply:
Good to know. But as this is like 3 lines I copied from another file I'm fine 😄